decisions/2021/03/02-fe-initiatives.md

```diff
@@ -0,0 +1,122 @@
+Date: 2021-03-02
+
+Attending: Austin, Birk, Bjørn, Christos, Debora, Edoardo, Ismay, Jan
+Hendrik, Jan-Gerke, Jennifer, Joakim, Joe, Martin, Médi-Rémi, Milagros,
+Viktor
+
+## Notes
+
+### Dynamic modules
+
+Coming along nicely, based on SystemJS. Demo-time if there is time!
+
+### GitHub Actions stuck in "queued"
+
+GitHub needs to manually move our workflow runs that are stuck in a
+queued phase to a completed phase. Then we can re-run them and get
+the 2.35.2 release built.
+
+### Working group updates
+
+#### Developer portal
+
+Revamped developer portal is live!
+
+All content has been ported over.
+
+Will start adding new content now.
+
+One-stop-shop for DHIS2 developer needs, more guides and tutorials are
+coming.
+
+Added the core developers with access rights to the
+dhis2/developer-portal repo.
+
+#### Cypress
+
+Add network shim to an existing app, didn't work perfectly.
+
+Network shim doesn't support static fixtures defined manually, needs to
+be fixed in the shim, and is being worked on.
+
+#### UI & Platform
+
+Tree-shakable library builds are now available. Now we only transpile
+our sources, which makes it easier to tree-shake in the application.
+
+In UI we've run into some problems related to imports in Storybook.
+
+#### App Hub and App Management updates
+
+App Hub taken a bit of a backseat to 2.36 fixes.
+
+Some deployment issues on AWS EBS.
+
+App Management changes relate to the ability to publish to the App Hub
+from our development pipelines, so we can leverage the bundled app
+override mechanisms. 
+
+Faster deployments to the field.
+
+### Topics
+
+#### PR previews
+
+Using a GitHub Action to build and publish a built artifact to Netlify.
+
+The artifact is then pushed to Netlify and added to the PR as a
+deployment.
+
+Surge is an alternative to Netlify when it comes to PR previews.
```

Comment:
Oh and https://vercel.com is the alternate service I was talking about @paschalidi (forgot that they changed their name).